### PR TITLE
Autocomplete for forms

### LINKED ISSUE #7: Forms: Type ahead
Whenever someone inputs a new person, event, org, etc., send AJAX requests to database for autocompletion/suggestions....

### PR DESCRIPTION
**Pull request by evz**
Closes #7

### CODE REVIEW

File: .cache/v/cache/lastfailed [hunk 1]
```diff
@@ -0,0 +1 @@
+{}
```
Comment (evz) [inline .cache/v/cache/lastfailed]: This seems like something we might not want under version control. Actually, what the heck is the `.cache` folder anyways?

File: occrp/templates/story.html [hunk 2]
```diff
@@ -125,3 +125,52 @@
...
+              }
+
+              response(suggestions);
+            });
+        }
+    };
+
+    function autocomplete_select() {
+        return function( event, ui ) {
+          console.log("Selected: " + ui.item.value);
```
Comment (evz) [inline occrp/templates/story.html]: Did you mean to do something other than logging out the value here?

File: occrp/views.py [hunk 3]
```diff
@@ -158,6 +159,27 @@ def about():
...
+    term = request.args['q']
+
+    return autocomplete_results(Organization.name, term)
+
+
+@views.route('/source-autocomplete/')
+def source_autocomplete():
+    term = request.args['q']
+
+    return autocomplete_results(Source.label, term)
```
Comment (evz) [inline occrp/views.py:180]: This is a very nice solution.

File: .cache/v/cache/lastfailed (hunk 1 above)
Comment (reginafcompton) [reply .cache/v/cache/lastfailed]: Hmmmm....I don't know. It does not seem to be meaningful, so I'll remove it.

File: occrp/templates/story.html (hunk 2 above)
Comment (reginafcompton) [reply occrp/templates/story.html]: No, I wanted that for testing...but it's useless. I'll clean it out.

**Pull request merged**